source/reference/verification.txt

```diff
@@ -16,117 +16,103 @@ Verify Data Transfer
-source cluster to the destination cluster.
-
-While hash comparison ensures that the destination cluster has received all
-changes from the source, the :dbcommand:`dbHash` command locks the cluster,
-preventing additional writes until it completes.
-
-.. note::
-
-   Hash comparison is not possible with sharded clusters. It also does not
-   work for standalone servers and replica sets that use MongoDB 4.4 or earlier
-   releases, since the document field order can vary.
-
-For more information, see :ref:`c2c-verify-hash-comp`.
-
-.. _c2c-verify-with-doc-comp:
-
-Document Comparison
-~~~~~~~~~~~~~~~~~~~
-
-You can verify sync by comparing documents on the source and destination
-clusters. Write a script that queries collections on the source cluster and
-then checks that the document exists with the same values on the destination
-cluster.
-
-.. _c2c-verify-with-verifier:
-
-Migration Verifier
-~~~~~~~~~~~~~~~~~~
-
-Migration Verifier connects to the source and destination clusters and performs
-a series of verification checks, comparing documents, views, and indexes to
-confirm the sync was successful.
-
-Migration Verifier does not support DDL operations. Do not run any DDL 
-operations on the source cluster while verifying data with Migration Verifier. 
-
-.. important::
-
-   Migration Verifier is an experimental and unsupported tool.
-
-   For installation instructions and usage limitations, see
-   `GitHub <https://github.com/mongodb-labs/migration-verifier>`__.
-
-Unlike other verification methods, Migration Verifier can run concurrently
-with ``mongosync``, checking documents on the destination cluster as they
-sync.
-
-For more information, see :ref:`c2c-migration-verifier`.
+-----
+
+You should verify your data after every sync. This is important
+in cases where you plan to move your application load from the
+source to the destination cluster.
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Verification Method
+     - Description
+   * - :ref:`Embedded Verifier <c2c-embedded-verifier>`
+     - Starting in 1.9, ``mongosync`` includes an embedded
+       verifier, which runs a series of verification checks on
+       the source and destination clusters to ensure a
+       successful sync. This is the preferred verification
+       method for deployments that meet the :ref:`requirements
+       <c2c-verify-embedded-limitations>`.
+
+       When the ``mongosync`` process starts, it prompts the
+       user with a disclaimer about the embedded verifier. You
+       must accept the disclaimer, or if you have already read
+       and acknowledged it, start ``mongosync`` with the
+       :option:`--acceptDisclaimer` option to use the verifier.
+
+       The verifier doesn't check every aspect of a migration.
+       To verify sync for unsupported collections, indexes, and
+       metadata, use a different verification method. 
+
+   * - :ref:`Document Counts <c2c-verify-doc-counts>`
```

Comment:
I realize we took away the note about mongosync needing to fully commit before calling any of these strategies 1) document counts 2) dbHash 3) document comparison script - can we note that mongosync must fully commit in each of these descriptions?

Reply:
Fixed